### PR TITLE
Prototype fixes

### PR DESCRIPTION
**Pull request by mac2648**
reduce jump height
reduce all the speed of rat and player
gargoyle AI bug fixes
added the ability to keep gates open if a specific interactable is on
destroy the body after consuming
some fine tunning of the vision for the AIs
added foliage assets
added foliage to the level
added small height differences for terrain in map

### CODE REVIEW

**Review by Entwistle-905 — APPROVED** [review]
thank you for your hard work d^-^b

File: Source/Redemption/Public/Gate.h [hunk 1]
```diff
@@ -17,6 +17,10 @@ class REDEMPTION_API AGate : public AActor
 	UPROPERTY(EditAnywhere)
 	TMap<AInteractable*, bool> InteractableKeys;
 
+	//when this interactable is active the gate will always be open does not matter the InteractableKeys
```
Comment (Fedic) [inline Source/Redemption/Public/Gate.h:20]: Huge bonus for puzzle creation

File: Source/Redemption/Private/Gate.cpp [hunk 2]
```diff
@@ -54,14 +59,24 @@ void AGate::CheckOpenConditions()
...
+	{
+		if (MasterKey->GetIsActive())
+		{
+			ShouldOpen = true;
+		}
+	}
+
 	if (ShouldOpen && !IsOpen)
 	{
+		//SOUND -> play here
```
Comment (Fedic) [inline Source/Redemption/Private/Gate.cpp:72]: Thank you for including these

**Review by thakkarnandish — APPROVED** [review]
Excellent work, thanks for all the hard work :)

**Review by Fedic — APPROVED** [review]

**Pull request merged**